src/headers/syscheck_op.h

```diff
@@ -82,6 +82,7 @@
 typedef enum fim_fields {
     FIM_FILE,
     FIM_HARD_LINKS,
+    FIM_MODE_FIELD,
```

Comment:
We should remove `_FIELD` from the name.

Reply:
Done 612cc75